client/signup/processing-screen/index.jsx

```diff
@@ -150,15 +152,17 @@ export class SignupProcessingScreen extends Component {
 		);
 	}
 
-	render() {
-		/* eslint-disable wpcalypso/jsx-classname-namespace */
+	componentDidUpdate = () => {
```

Comment:
Not a problem, but any reason to bind to `this`?

Reply:
Actually there is nothing wrong with using the traditional `function` syntax here, but I personally found the fat arrow syntax quite charming. Thus it's just a personal preference 👻

Reply:
It is quite charming! And most likely not a performance issue at all in this case. Just noting though that in general a traditional `function` syntax translates roughly to `SignupProcessingScreen.prototype.componentDidUpdate = function(){ .. }` and thus just one `function` object is created regardless of number of instances of `SignupProcessingScreen` we create. A fat arrow notation translates to something like
```js
function SignupProcessingScreen() { 
    this.componentDidUpdate = (function(){ .. }).bind(this);
}
```
Thus a new function object is created for every instance of `SignupProcessingScreen`. Which again - is not a problem at all in this case. Just something to keep in mind :-)